app/src/main/kotlin/com/wire/android/di/KaliumConfigsModule.kt

```diff
@@ -69,7 +69,7 @@ class KaliumConfigsModule {
             wipeOnCookieInvalid = BuildConfig.WIPE_ON_COOKIE_INVALID,
             wipeOnDeviceRemoval = BuildConfig.WIPE_ON_DEVICE_REMOVAL,
             wipeOnRootedDevice = BuildConfig.WIPE_ON_ROOTED_DEVICE,
-            isWebSocketEnabledByDefault = !context.isGoogleServicesAvailable()
+            isWebSocketEnabledByDefault = BuildConfig.WEBSOCKET_ENABLED_BY_DEFAULT || !context.isGoogleServicesAvailable()
```

Comment:
can we remove the !context.isGoogleServicesAvailable() ?

Reply:
Is part of the requirement, we need to enable it by default if no Google play services available.
But the `BuildConfig` has preference if true